src/main/java/dev/openfeature/javasdk/EvaluationContext.java

```diff
@@ -1,139 +1,31 @@
 package dev.openfeature.javasdk;
 
-import dev.openfeature.javasdk.internal.Pair;
-import lombok.*;
```

Comment:
Similar to above, either make your editor not do these formatting changes or check in something that consistently applies it.